.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# Legal
+/LICENSE  @getsentry/owner-legal
+
+# Migrations
+south_migrations/  @getsentry/owner-migrations
+
+# Snuba
+/src/sentry/eventstream/        @getsentry/owner-snuba
+/src/sentry/utils/snuba.py      @getsentry/owner-snuba
+/src/sentry/tsdb/snuba.py       @getsentry/owner-snuba
+/src/sentry/tsdb/redissnuba.py  @getsentry/owner-snuba
+/src/sentry/search/snuba/       @getsentry/owner-snuba
+/src/sentry/tagstore/snuba/     @getsentry/owner-snuba
+
+# Ted stuff
```

Comment:
So I know we'd want to leverage teams for everything, but certain chunks of codebase are clearly owned by one or two people. So opinions on how crazy to go with teams? I could go crazy and be like, `getsentry/owner-{similarity,digests,lua} or somethign and replace all the `@tkaemming` rules here, but that seems heavy handed to create a ton of groups.

Reply:
In my opinion: personally assigning to individuals is going to be pretty realistic in a lot of these areas (or at least reflects my own personal understanding today — I think you, David, and Armin are going to have large chunks in here as well) and I don't think that should be hidden behind a single member team.

Reply:
(…and if requiring owner review on files/directories where there is only an individual owner is going to be a problem, I think that is a larger issue.)

Reply:
I think it makes sense for a good chunk of stuff too. And this might even help be some indicator of bus factor. "Why does all of this have 1 person responsible?" I'd even argue going to 2 people is fine, and beyond that move to a group.

I thikn there are some obviously clear ones like Snuba where the team is only two people, but is very distinctly a team. Even if the team is small. Other chunks are much more... miscellaneous with no real owner, whether through history or just practicality. And realistically some of these things _are_ just blocked behind one person unfortunately.